README.md

```diff
@@ -1,51 +1,135 @@
 
-This example demonstrates how to run one validator client and one beacon node using [Prsym](https://docs.prylabs.network/docs/getting-started) in a single node kubernetes with [kind](https://kind.sigs.k8s.io/). This setup uses `hostPath` to store beacon and validator's data with the host mount added on the kind node. Please note this configuration is for testing only.
+**This project is still in development and it's NOT recommended to be used on mainnet.**
+
+## Install Prysm beacon node and validator with Helm
+
+If the goal is to run Prysm on mainnet, we recommend to use
+
+- Production-grade k8s distribution to build a k8s cluster.
+- [NFS](https://en.wikipedia.org/wiki/Network_File_System) as the persistent storage.
+- [Helm](https://helm.sh/) to manage packages and releases.
 
 ### Prerequisite
 
-* [kind](https://kind.sigs.k8s.io/)
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Install a k8s distribution and build a cluster.
+- [Install Helm](https://helm.sh/docs/intro/install/) on the k8s controller node.
+
+### Install and configure NFS
+
+1. Set up NFS (Example: [Guide for NFS installation and configuration on Ubuntu](https://ubuntu.com/server/docs/service-nfs)).
+
+2. Create beacon node and validator data folders with the correct ownership (we use uid 1001 and gid 2000) on NFS.
 
-### Prepare the manifests
+3. Create the wallet folder on NFS and import validator accounts (Example: [Import your validator accounts into Prysm](https://docs.prylabs.network/docs/mainnet/joining-eth2#step-4-import-your-validator-accounts-into-prysm)).
+
+4. Export created data and wallet folders as described in the NFS configuration guide.
+
+### Change the configurations to match your environment
 
 1. Clone this repo.
-    ```
+
+    ```bash
     git clone https://github.com/eth2xk8s/eth2xk8s.git
     ```
 
-2. Create the data folders for beacon node and validator on the host machine. For example:
-    ```
-    mkdir -p /data/prysm/validator /data/prysm/beacon
-    ```
+2. Change values in `./eth2prysm/values.yaml`.
 
-3. Update the `extraMounts` in `cluster-config/kind-single-node.yaml` with the path to the created data folder.
+   It's recommended to read through each field in `values.yaml` and determine the desired values. Fields that need to be changed or verified before installing the chart are the following ones:
 
-4. Create the validator wallet and replace `example-password` with your wallet password in `host-path/wallet-secret.yaml`.
+   - **nfs.serverIp**: NFS server IP address.
+   - **image.version**: Prysm client version.
+   - **beacon.dataVolumePath**: The beacon node data volume path/folder on NFS.
```

Comment:
The path to the data directory on the NFS for beacon node.

Reply:
Changed. Also changed the descriptions for `validators.validator1.dataVolumePath` and `validators.validator1.walletVolumePath`.